blazor/diagram/ports/interaction.md

```diff
@@ -13,7 +13,7 @@ The port can be used to create connector by enabling the `Draw` in the [PortCons
 
 ## How to draw connector from the port
 
```

Comment:
You can draw connectors from the port by enabling Draw constraints to the Constraints property. By default, the connector segment type is set to [Orthogonal](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.ConnectorSegmentType.html#Syncfusion_Blazor_Diagram_ConnectorSegmentType_Orthogonal).